src/develop/pixelpipe_hb.c

```diff
@@ -1301,6 +1295,7 @@ static int dt_dev_pixelpipe_process_rec(dt_dev_pixelpipe_t *pipe, dt_develop_t *
     if(dt_opencl_is_inited() && pipe->opencl_enabled && pipe->devid >= 0)
     {
       int success_opencl = TRUE;
+      int input_cst_cl = input_format->cst;
```

Comment:
int -> dt_iop_colorspace_type_t ?

Reply:
This happens a lot, I'll fix it as I find them.